rocketmq-spring-boot/src/main/java/org/apache/rocketmq/spring/support/RocketMQUtil.java

```diff
@@ -326,4 +326,10 @@ public static DefaultLitePullConsumer createDefaultLitePullConsumer(String nameS
 
         return litePullConsumer;
     }
+
+    public static String getNamespace(String specifiedNamespace, String defaultNamespace) {
```

Comment:
if specifiedNamespace and defaultNamespace are both null. it need not to set name space to container.

Reply:
it's just for convenience, if rocketmq.producer.namespace or rocketmq.pushconsume.namespace is specified, then we don't need to specify in each producer or consumer. if they are both null, it has no effect.